src/test/java/com/spotify/confidence/TypeMapperTest.java

```diff
@@ -0,0 +1,108 @@
+package com.spotify.confidence;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.google.protobuf.util.Values;
+import dev.openfeature.sdk.MutableStructure;
+import dev.openfeature.sdk.Value;
+import dev.openfeature.sdk.exceptions.ValueNotConvertableError;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+
+class TypeMapperTest {
+
+  @Test
+  public void testFromIntegerNumber() {
+    // given
+    final Value intValue = new Value(42);
+    // when
+    final com.google.protobuf.Value protoValue = TypeMapper.from(intValue);
+    // then
+    assertEquals(42, protoValue.getNumberValue());
+  }
+
+  @Test
+  public void testFromDoubleNumber() {
+    // given
```

Comment:
Nit: could you update the comments to this test to keep the test comments consistent?